dts/bindings/sensor/st,stm32-vbat.yaml

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 family VBAT node
+
+include: base.yaml
+
+compatible: "st,stm32-vbat"
+
+properties:
+    io-channels:
+      required: true
+      description: ADC channel for Vbat sensor
+
+    ratio:
+      type: int
+      default: 3
+      description: fraction of VBat to be connected to the ADC input
+
+    voltage-mv:
+      type: int
+      default: 3300
+      description: Vbat voltage reference in millivolts
```

```suggestion
    vref-mv:
      type: int
      default: 3300
      description: ADC voltage reference in millivolts
```

This voltage reference is the same for all inputs of that ADC, isn't it? Should it then not go into the ADC node?

---

yes, actually it is the same as the vref_mv property  of the st,stm32-adc.yaml